packages/manager/modules/navbar/src/notifications-menu/template.html

```diff
@@ -8,15 +8,167 @@
+                        data-oui-tooltip="{{::menuLink.isActive ?
+                            $ctrl.translations.notification.markRead :
+                            $ctrl.translations.notification.markUnread
+                        }}"
+                    >
+                        <span
+                            class="oui-icon oui-icon-shape-dot"
+                            data-ng-if="menuLink.isActive"
+                            aria-hidden="true"
+                        ></span>
+                        <span
+                            class="oui-icon oui-icon-shape-circle"
+                            data-ng-if="!menuLink.isActive"
+                            aria-hidden="true"
+                        ></span>
+                    </button>
+                </div>
+            </li>
+            <li
+                class="oui-navbar-menu__item oui-navbar-notification oui-navbar-notification-placeholder"
+                data-ng-if="$ctrl.sublinks.length === 0"
+            >
+                <div class="oui-navbar-notification-placeholder__content">
+                    <span
+                        class="oui-icon oui-icon_bicolor oui-icon-success_circle oui-navbar-notification__icon"
+                        aria-hidden="true"
+                    >
+                    </span>
+                    <div
+                        class="oui-navbar-notification-placeholder__title"
+                        data-ng-bind="::$ctrl.translations.notification.noNotification"
+                    ></div>
+                    <div
+                        class="oui-navbar-notification-placeholder__description"
+                        data-ng-bind="::$ctrl.translations.notification.noNotificationDescription"
+                    ></div>
+                </div>
+            </li>
+            <li
+                class="oui-navbar-menu__item oui-navbar-notification oui-navbar-notification-placeholder"
+                data-ng-if="!$ctrl.sublinks"
+            >
+                <div class="oui-navbar-notification-placeholder__content">
+                    <span
+                        class="oui-icon oui-icon_bicolor oui-icon-error_circle oui-navbar-notification__icon"
+                        aria-hidden="true"
+                    >
+                    </span>
+                    <div>
+                        <div
+                            class="oui-navbar-notification-placeholder__title"
+                            data-ng-bind="::$ctrl.translations.notification.errorInNotification"
+                        ></div>
+                        <div
+                            class="oui-navbar-notification-placeholder__description"
+                            data-ng-bind="::$ctrl.translations.notification.errorInNotificationDescription"
+                        ></div>
+                    </div>
+                </div>
+            </li>
+        </ul>
```

Comment:
Wasn't this already handled by the navbar ? Or some custom behaviour was required ?

Reply:
custom behaviour was required and Axel recommended to do this way. This way old component also works as it is.